modules/globebrowsing/tile/tile.h

```diff
@@ -109,7 +99,7 @@ class Tile : public cache::MemoryAwareCacheable {
     static const Tile TileUnavailable;
 
 private:
-    std::shared_ptr<ghoul::opengl::Texture> _texture;
+    ghoul::opengl::Texture* _texture;
```

Comment:
We are using a kind of null-tile `static const Tile TileUnavailable` which has a `nullptr` texture. At this point it is required since tile providers must be able to return tiles. It could use some place holder texture I guess but I think it makes sense to have pointers which can be null.